pr-deploy/package.json

```diff
@@ -21,15 +22,14 @@
     "@google-cloud/storage": "3.0.2",
     "dotenv": "8.0.0",
     "express": "4.17.1",
-    "fancy-log": "1.3.3",
```

Comment:
We ain't fancy no more? 😢

Reply:
Of course we are :nail_care: just no need to log things anymroe